tools/test.py

```diff
@@ -21,13 +21,15 @@ def single_gpu_test(model, data_loader, show=False):
     results = []
     dataset = data_loader.dataset
     prog_bar = mmcv.ProgressBar(len(dataset))
+    score_thr = model.module.test_cfg['score_thr']
+
     for i, data in enumerate(data_loader):
         with torch.no_grad():
-            result = model(return_loss=False, rescale=not show, **data)
+            result = model(return_loss=False, rescale=False, **data)
```

Comment:
If `show` is specified to False, which means the results are not used for debugging or demonstration, then we need to rescale the results to the original size.